subprojects/core/src/main/java/org/gradle/testfixtures/internal/TestBuildRootBuildOperationType.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.testfixtures.internal;
+
+import org.gradle.internal.operations.BuildOperationType;
+
+public interface TestBuildRootBuildOperationType extends BuildOperationType<TestBuildRootBuildOperationType.Details, TestBuildRootBuildOperationType.Result> {
```

Comment:
Seems like some of changes are not related to your PR, right?
Could you fix it? Let me know if you'll have a problems with it